scripts/post_model.js

```diff
@@ -0,0 +1,185 @@
+/* eslint-disable no-unused-vars */
+/**
+ * PostModel class
+ */
+class PostModel {
+  _posts = [];
+
+  _ids = new Set();
+  _standartFilter = {
```

Comment:
See: you do not use context of one instance of PostModel, _standartFilter is the same for all of instances, so in that case it should be static.
'+ Cosmetic: most likely we expect it will not change so potentially it is a constant (basically in that case we can assume it public, so simple CONSTANT naming notation will be enough).